lib/yt/collections/reports.rb

```diff
@@ -61,6 +62,12 @@ class Reports < Base
         mobile: 'MOBILE' # only present for data < September 10, 2013
       }
 
+      # @see https://developers.google.com/youtube/analytics/v1/dimsmets/dims#Playback_Detail_Dimensions
+      PLAYBACK_DETAILS = {
```

Comment:
Can you rename to `SUBSCRIBED_STATUSES`? 

"Playback details" include other things like live or on demand etc…